README.md

```diff
@@ -24,11 +24,11 @@ Using Gradle makes building WPILib very straightforward. It only has a few depen
 ## Requirements
 
 - A C++ compiler
-    - On Linux, gcc works fine
+    - On Linux, `gcc` works fine
```

Comment:
In this context, gcc is the name of the compiler not a piece of code.

Reply:
If, contextually, it's being used as the name of the compiler, then shouldn't it be GCC?  Keep in mind that `gcc` is also the command to invoke GCC.